src/com/palmergames/bukkit/towny/TownyMessaging.java

```diff
@@ -93,22 +93,19 @@ public static void sendMsg(String msg) {
 	 * and to the named Dev if DevMode is enabled.
 	 * Uses default_towny_prefix
 	 *
-	 * @param sender the Object sending the message
+	 * @param sender the CommandSender receiving the msg
 	 * @param msg the message being sent
 	 */
-	public static void sendMsg(Object sender, String msg) {
+	public static void sendMsg(CommandSender sender, String msg) {
 		if (sender != null) {
-			if (sender instanceof Resident) {
-				Player p = TownyAPI.getInstance().getPlayer((Resident) sender);
-				if (p != null) {
-					p.sendMessage(Translation.of("default_towny_prefix") + ChatColor.GREEN + msg);
-				}
+			if (sender instanceof Player) {
+				Player p = (Player)sender;
+				p.sendMessage(Translation.of("default_towny_prefix") + ChatColor.GREEN + msg);
 			} else {
-				CommandSender toSend = (CommandSender) sender;
-				if (toSend instanceof ConsoleCommandSender) {
-					toSend.sendMessage(ChatColor.stripColor(msg));
+				if (sender instanceof ConsoleCommandSender) {
+					sender.sendMessage(ChatColor.stripColor(msg));

```

Comment:
Would probably look neater if the `instanceof` conditionals were chained together. E.g:
```java
if (sender instanceof Player) {
...
} else if (sender instanceof ConsoleCommandSender) {
...
} else {
...
}
```